fieldservice/models/fsm_location.py

```diff
@@ -33,7 +34,6 @@ def _tz_get(self):
                                  domain="[('is_company', '=', False),"
                                         " ('fsm_location', '=', False)]",
                                  index=True)
-    tag_ids = fields.Many2many('fsm.tag', string='Tags')
```

Comment:
@osi-scampbell Really? No tag at all?

Reply:
Correct, we aren't currently using tag_ids because res.partner already has a field 'category_id' that is a man2many field for "Tag Ids." That was just a duplicate field that was only being referenced on the search view at line 167 which I switched with category_id.

Reply:
Ok, so `fsm.tag` are only used on `fsm.order`, right?

Reply:
That is correct.